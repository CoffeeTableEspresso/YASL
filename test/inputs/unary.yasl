##-10\n4\nfalse\ntrue\n-1\n

echo -10
echo len'YASL'
echo !true
echo !false
echo ^0b00000000
