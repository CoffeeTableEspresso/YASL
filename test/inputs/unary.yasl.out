-10
4
false
true
-1
